synapse/lib/stormlib/cortex.py

```diff
@@ -1028,7 +1028,9 @@ async def _methReply(self, code, headers=None, body=s_stormtypes.undef):
         if body is not s_stormtypes.undef:
             if not isinstance(body, bytes):
                 body = await s_stormtypes.toprim(body)
-                body = json.dumps(body).encode('utf-8', 'surrogatepass')
+                # FIXME: handle data which has surrogate issues
```

Comment:
Discuss how to fix this.

Reply:
We allow storing `str` types in nodes that have surrogate issues ( see [StorTypeUtf8](https://github.com/vertexproject/synapse/blob/v2.197.0/synapse/lib/layer.py#L551-L611) ), so we definitely should be able to handle serializing them via json in & out. If we don't have tests for pushing those values over HTTP APIs, we definitely should.